### PR TITLE
JIT: avoid LCL_FLD stress for locals that already have LCL_FLD appear…

### LINKED ISSUE #76855: Test failure: JIT/opt/Cloning/Runtime_61040_5/Runtime_61040_5.sh
linux-x64 random JitStress

https://dev.azure.com/dnceng-public/public/_build/results?buildId=46805&view=ms.vss-test-web.build-test-results-tab&runId=941484&resultId=110071&paneView=dotnet-dnceng.dnceng-build-release-tasks.helix-test-information-tab

```
export COMPlus_TieredCompilation=0
export COMPlus_JitStress=281
...
    JIT/opt/Cloning/Runtime_61040_5/Runtime_61040_5.sh [FAIL]
      
      Return code:      1
      Raw output file:      /datadisks/disk1/work/AFE709AC/w/B0110972/uploads/Reports/JIT.opt/Cloning/Runtime_61040_5/Runtime_61040_5.output.txt
      Raw output:
      BEGIN EXECUTION
      /datadisks/disk1/work/AFE709AC/p/corerun -p System.Reflection.Metadata.MetadataUpdater.IsSupported=false Runtime_61040_5.dll ''
      Expected: 100
      Actual: 102
      END EXECUTION - FAILED
      Test Harness Exitcode is : 1
      To run the test:
      > set CORE_ROOT=/datadisks/disk1/work/AFE709AC/p
      > /datadisks/disk1/work/AFE709AC/w/B0110972/e/JIT/opt/Cloning/Runtime_61040_5/Runtime_61040_5.sh
      Expected: True
      Actual:   False
      Stack Trace:
           at JIT_opt._Cloning_Runtime_61040_5_Runtime_61040_5_._Cloning_Runtime_61040_5_Runtime_61040_5_sh()
           at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
           at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

Failure is in:
```
private static void ProblemWithLclFldAsg(int[] arr, int idx)
{
    for (int i = 0; i < ArrLen; i++)
    {
        *(byte*)&i = (byte)idx;
        arr[i] = 0;
    }
}
```

Also fails on win-x64.

@dotnet/jit-contrib

### PR DESCRIPTION
**Pull request by AndyAyersMS**
…ances

Also lay the groundwork for making this stress a bit more random.

Fixes #76855.

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
@BruceForstall PTAL
cc @dotnet/jit-contrib

**Comment by AndyAyersMS** [comment]
Failures are unrelated (this is a jit stress only fix).

**Comment by AndyAyersMS** [comment]
Jit stress failures are known issue #76880.

**Comment by AndyAyersMS** [comment]
@BruceForstall ping

File: src/coreclr/jit/lclvars.cpp [hunk 1]
```diff
@@ -8286,6 +8287,13 @@ Compiler::fgWalkResult Compiler::lvaStressLclFldCB(GenTree** pTree, fgWalkData*
 
     if (bFirstPass)
     {
+        // Ignore locals that alreay have field appearances
```
Comment (BruceForstall) [inline src/coreclr/jit/lclvars.cpp:8290]: typo
```suggestion
        // Ignore locals that already have field appearances
```

**Review by BruceForstall — APPROVED** [review]

File: src/coreclr/jit/lclvars.cpp (hunk 1 above)
Comment (AndyAyersMS) [reply src/coreclr/jit/lclvars.cpp:8290]: My spelling seems to be regressing recently. 

Hate to have to retrigger CI just to fix things like this.

**Pull request merged**